Article/Article.js

```diff
@@ -33,6 +33,21 @@ class Article {
     } else {
       this.expandButton.textContent = "expand";
     }
+    var headerElement = $(".header");
```

Comment:
Neat little trick built into the Chrome DevTools is that you can use `$` and `$$` in place of `document.querySelector`.and `document.querySelectorAll`.